task-2/webpack.config.js

```diff
@@ -30,6 +30,10 @@ module.exports = {
           },
         },
       },
+      {
+        test: /\.css$/,
+        use: [{ loader: 'style-loader' }, { loader: 'css-loader' }],
```

Comment:
Feel free to add a SCSS support.

Reply:
SASS support has been added. 
Ready to use.